src/components/Footer/Footer.jsx

```diff
@@ -1,5 +1,11 @@
+import classes from './Footer.module.scss';
+
 function Footer() {
-  return <div>Footer</div>;
+  return (
+    <footer className={classes.footer}>
+      <div className={classes.copyright}>Project by Gleb Dronchenko, IsSoft Solutions, 2021</div>
```

Comment:
i think it will be better tu put this string inside a const

Reply:
I have deleted extra tag.